ga/src/main/java/nl/tudelft/serg/evosql/metaheuristics/StandardGA.java

```diff
@@ -87,57 +87,53 @@ public Fixture execute(long pathTime) throws SQLException {
 				&& System.currentTimeMillis() - startTime < pathTime
 				//&& generations < maxGenerations
 		){
-			List<Fixture> offsprings = new ArrayList<Fixture>(populationSize);
+			List<Fixture> offspring_population = new ArrayList<Fixture>(populationSize);
 			for (int index=0; index < populationSize; index += 2){
 				// Get two parents through selection operator
 				Fixture parent1 = selection.getFixture(population);
 				Fixture parent2 = selection.getFixture(population);
 
 				Fixture offspring1;
 				Fixture offspring2;
-				boolean o1Changed = false, o2Changed = false;
 				
 				// Use the crossover operator
-			    if (crossover.canBeDone(parent1, parent2) && random.nextDouble() < EvoSQLConfiguration.P_CROSSOVER) {
+			    if (random.nextDouble() < EvoSQLConfiguration.P_CROSSOVER) {
 
-					List<Fixture> generatedOffsprings = this.crossover.crossover(parent1, parent2);
-					offspring1 = generatedOffsprings.get(0);
-					offspring2 = generatedOffsprings.get(1);
+					Fixture[] generatedOffspring = this.crossover.crossover(parent1, parent2);
+					offspring1 = generatedOffspring[0];
+					offspring2 = generatedOffspring[0];
```

Comment:
Why do we only use the first generated offspring and not both?

Reply:
Right! It should be fixed